.evergreen/run-tests.sh

```diff
@@ -144,5 +144,5 @@ else
               ${MULTI_MONGOS_URI_SYSTEM_PROPERTY} ${API_VERSION} ${GRADLE_EXTRA_VARS} ${ASYNC_TYPE} \
               ${JAVA_SYSPROP_NETTY_SSL_PROVIDER} \
               -Dorg.mongodb.test.fle.on.demand.credential.test.failure.enabled=true \
-              --stacktrace --info --continue test
+              --stacktrace --info --continue test integrationTest
```

Comment:
This was intentionally omitted here since it only affect Scala and the extra test coverage provided by running these tests against every server version was not considered beneficial enough to be worth it.  Can we get the coverage we need with Scala-specific tests?

Reply:
Done. Also updated https://github.com/mongodb/mongo-java-driver/pull/1326#discussion_r1511881774 correspondingly.